pkg/services/authn/authnimpl/registration.go

```diff
@@ -106,6 +114,11 @@ func ProvideRegistration(
 	rbacSync := sync.ProvideRBACSync(accessControlService)
 	if features.IsEnabledGlobally(featuremgmt.FlagCloudRBACRoles) {
 		authnSvc.RegisterPostAuthHook(rbacSync.SyncCloudRoles, 110)
+
+		_, exists := pluginRegistry.Plugin(context.Background(), cloudHomePluginId, "")
```

Comment:
We don't need this check. If the feature toggle is enabled then we should register the hook